### PR TITLE
Deploy to PyPI on release

### LINKED ISSUE #84: Create release on merge to master and deploy to PyPI
- Automatically cut a new release, populate changelog info and tag release when merging from development to master. 
- Add check that version has been changed. 
- Deploy release on PyPI with Twine.

### PR DESCRIPTION
**Pull request by peterk87**
Fixes #84 and #95

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,39 +1,29 @@
...
+    secure: feBsWxq8pSl/Eisu262EJH3E3JYgeWVmmwHCHut1474BxqXDvoUOGQmD7WoxwLCUVwvTE9gq37KZhYA4QDBwGI2vTwEegEeZGEo6ivPPO68TEkEFh00HwvXmFnYjllTtVISkUpBiUT8bYXdcPzR66bTfig+tZvlKIMATbHhzwFrWh1dkWkyiUojUaD4T861M0n774B0T8nyS4ppb7GT4CfkJXm9PpGMNWu04Cv0U2tTAIABT0OvvrstP1o1XFvOLNP3KI7aV/Onf24r3D5/jqtQN5ET90HFqvpoumRQi4XnJ10RWLySSOIlJ8HEE7giHvFz//TAGA4e3rBB4B0POun+qtAlriH2aq62vpBhm4jLZQOI8ARm6G1GPBKXZq+LnFJmQKFritYpNUubdpsaCK1ykoDgVOQh9FrAVXTcWvN6PPPwbrwZ2L5DkKsKyYK9QdtgM7Yhhr3cT2tM74jPQAeh9ky7iow4ltTlX/4NNegXwVxn7Gmn8Dkeg5dFeh/CcfaaHKSd3Se2DUe3vPldoQgkNJSUyFL65MvfftQP/pRRpLfPqkrHl6QjhCTdLLUvTTtyQtkvO5a3vNKtbScMK+epvGKrwSSPBIckYS0BUWHM/KoswnR+VPWh5wQmJWLHliukuseCx7qXJkABuZNsl8YtFXB+uCXGV9eoErR4sD5c=
+  on:
+    tags: true
+    repo: phac-nml/bio_hansel
+    python: 3.6
+notifications:
+  email:
+    recipients:
+    - mgopez95@gmail.com
+    - peter.kruczkiewicz@gmail.com
```
Comment (Takadonet) [inline .travis.yml]: Perhaps add myself and Darian on the notification list.

File: setup.py [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
-from setuptools import find_packages
+from setuptools import setup, find_packages
 
 from bio_hansel import __version__, program_name, program_desc
 
 classifiers = """
 Development Status :: 3 - Alpha
 Environment :: Console
-License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+License :: OSI Approved :: Apache Software License
```
Comment (Takadonet) [inline setup.py]: Why we change the license?

Comment (peterk87) [reply setup.py]: I noticed that the license in the setup.py wasn't the same as the License file and readme, which both say it's Apache 2.0. I think we want the license to be Apache? We could check with everyone else about the proper license for biohansel.

File: .travis.yml (hunk 1 above)
Comment (peterk87) [reply .travis.yml]: Added you and Darian's emails in caa77d81bc03f2d769714d7163a192ee7b2e09d0

File: setup.py (hunk 2 above)
Comment (Takadonet) [reply setup.py]: Good catch! Apache 2.0 is what other projects are.

**Pull request merged**